Cargo.toml

```diff
@@ -7,17 +7,15 @@ edition = "2018"
 license = "GPL-3.0-or-later"
 
 [features]
-ssh-agent = ["lnk-thrussh-agent", "lnk-thrussh-encoding"]
+ssh-agent = ["agent", "encoding"]
 
 [dependencies]
-async-trait = "0.1"
 byteorder = "1.4"
-futures = "0.3"
 generic-array = { version = "0.14", features = ["serde"] }
 lazy_static = "1"
-lnk-cryptovec = "0.6.0"
-lnk-thrussh-agent = { version = "0.1.0", optional = true, default-features = false }
-lnk-thrussh-encoding = { version = "0.1.0", optional = true }
+cryptovec = { version = "0.6.0", git = "https://github.com/radicle-dev/radicle-ssh" }
```

Comment:
You need to adjust the `deny.toml` with the new git sources.

Reply:
this should do https://github.com/radicle-dev/radicle-keystore/compare/bef4c29b28e3cd7c37e48b155665341bb6558cbe..7517325791f7d7a105dd5861e45b473f6ac4874e